debian/control

```diff
@@ -0,0 +1,29 @@
+Source: key-mapper
+Section: misc
+Priority: optional
+Maintainer: Sezanzeb <proxima@sezanzeb.de>
+Standards-Version: 4.5.0
+Build-Depends:
+ debhelper-compat (= 12),
```

Comment:
For a Debian package you should target the currently-recommended version:

```suggestion
 debhelper-compat (= 13),
```

If this packaging is supposed to target multiple releases, this might not be appropriate. It’s not an issue for Debian 10, `debhelper` 13 is available as a backport; it might be a problem for some releases of Ubuntu.

Reply:
Ubuntu 20.04 still has 12. I don't think we should depend on backport-packages if we can avoid it.

Reply:
That makes sense.